modules/ROOT/pages/tips-and-tricks.adoc

```diff
@@ -160,3 +160,21 @@ or setup shell `alias`es as needed to make them available to the CLI like so:
   $ alias evince="flatpak run org.gnome.Evince"
     # or alias evince="org.gnome.Evince"
   $ evince
+
+== How do to replace fedora-flatpak remote with flathub?
+. Enable the flathub remote if not already enabled using the instructions {docs-url}/getting-started/#flathub-setup[here]
```

Comment:
How does that interact with the filtered flathub view available by default? Maybe we should make sure to remove this one before?

Reply:
Yes, updating the getting started guide to get the unfiltered Flathub is probably better.